syntax="proto3";

message version_detail{ 
   string proj_id=5;
   repeated int32 change=2; 
   // 1=add 2=remove 3=update 
   repeated int32 line_number=3;
   repeated string line_change=4;
   repeated string change_time=6;
}

